lib/manager/npm/post-update/lerna.ts

```diff
@@ -44,10 +46,31 @@ export async function generateLockFiles(
       params =
         '--ignore-scripts --ignore-engines --ignore-platform --mutex network:31879';
     }
-    cmd = `npm i -g -C ~/.npm/lerna@${lernaVersion} lerna@${lernaVersion} && ${lernaClient} install ${params} && ~/.npm/lerna@${lernaVersion}/bin/lerna bootstrap --no-ci -- ${params}`;
-    if (binarySource === BinarySource.Global) {
-      cmd = `${lernaClient} install ${params} && lerna bootstrap --no-ci -- ${params}`;
+    if (config.binarySource === BinarySource.Docker) {
```

Comment:
Can we migrate this to use lib/util/exec ?

Reply:
Sure

Reply:
Was copy and paste from yarn / npm 🙈

Reply:
done